code/modules/sanity/inspiration_component.dm

```diff
@@ -72,6 +72,12 @@
 			if("Cognition")
 				stat_noun = pick("cognition", "intelligence", "knowledge")
 				stat_adjective = pick("smart", "intelligent", "concentrated", "knowledgable")
+			if("Vivification")
+				stat_noun = pick("invigoration", "vigorous", "healthy", "fit")
```

Comment:
invigoration -> invigorated

Reply:
Unless I'm wrong, the noun is generally used for oddities a la "this item feels somewhat like _____" wouldn't the best words for it be "vigor" "health" and "fitness", and just remove invigoration entirely?

Reply:
oh lmfao "invigoration" is the only noun in this list